### PR TITLE
[PRESS4-237]-[Sales Funnel License] Commerce Module YITH installation UI

### PR DESCRIPTION
**Pull request by sumathi3399**
JIRA ticket:- https://jira.newfold.com/browse/PRESS4-237

created the sales funnel license card in the store overview page.

Screenshot 2023-05-29 at 3 21 32 PM

### CODE REVIEW

File: src/configs/YITHPlugins.config.js [hunk 1]
```diff
@@ -8,6 +8,7 @@ import { ReactComponent as CustomizeAccount } from "../icons/yith-woocommerce-cu
 import { ReactComponent as Gift } from "../icons/yith-woocommerce-gift-card.svg";
 import { ReactComponent as WishList } from "../icons/yith-woocommerce-wishlist.svg";
 import { MarketplaceSdk } from "../sdk/marketplace";
+import { ReactComponent as InstallationUI } from '../icons/yith-woocommerce-installation.svg';
```
Comment (ashrithapradeep) [inline src/configs/YITHPlugins.config.js:11]: Can you rename the file. May be to `yith-sales-funnel.svg`

File: src/configs/YITHPlugins.config.js [hunk 2]
```diff
@@ -294,5 +295,30 @@ export const YITHPluginsDefinitions = (props) => ({
         },
       ],
     },
+    {
+      Card: FeatureCard,
+      shouldRender: () => true,
+      name: "",
```
Comment (ashrithapradeep) [inline src/configs/YITHPlugins.config.js:301]: You can add name, its not necessary that name should be same as slug

File: src/configs/YITHPlugins.config.js (hunk 1 above)
Comment (sumathi3399) [reply src/configs/YITHPlugins.config.js]: done

File: src/configs/YITHPlugins.config.js (hunk 2 above)
Comment (sumathi3399) [reply src/configs/YITHPlugins.config.js]: added the name

**Review by aulisius — APPROVED** [review]

**Pull request merged**